### DISCUSSION TITLE
Django 4.2 is supported?

### DISCUSSION THREAD

**Discussion by dlintin** [wagnerdelima/drf-social-oauth2#189] [General]
Hi, is the library support Django 4.2? I cant find any info regarding this anywhere.

**Comment by ylmzfurkan** [comment]
Hii @dlintin 

According to https://www.djangoproject.com/, the latest version of Django is 4.2.2. It was released on June 5, 2023. It is a Long-Term Support (LTS) release, which means that it will receive security and data-loss fixes for a guaranteed period of three years.

The previous LTS release was Django 3.2.11, which was released on March 8, 2023. It is supported until March 8, 2026.

If you are looking for the latest development version of Django, you can get it from the Django GitHub repository. The development version is not as stable as the LTS releases, but it may contain new features and bug fixes that are not yet available in the LTS releases.

**Reply by ylmzfurkan** [reply]
I wish I was a bot, but I'm not
